tests/integ/modin/test_plot.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import io
+
+import modin.pandas as pd
+import pandas as npd
+
+species = ["adelie"] * 3 + ["chinstrap"] * 3 + ["gentoo"] * 3
+measurements = ["bill_length", "flipper_length", "bill_depth"] * 3
+values = [37.3, 187.1, 17.7, 46.6, 191.7, 17.6, 45.5, 212.7, 14.2]
+
```

Comment:
Can we add a test where to plot a Series/DataFrame is passed?

According to https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html it seems to allow in `*args` a dataframe (and likely series) object. Just to make sure the frontend handles a necessary conversion if needed. 

It may also be that the documentation here is not accurate, or *args has self keyword folded. Prob. this scenario can be easily ticked off.

Reply:
The docs are not accurate. The semantics of these accessors is a little odd.